driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -224,27 +268,7 @@ public SimpleStatement setRoutingKey(ByteBuffer... routingKeyComponents) {
         return this;
     }
 
-    /**
-     * <p/>
```

Comment:
I would have kept the javadocs for internal reference, even if the method is now private.